server/app.js

```diff
@@ -118,24 +120,46 @@ app.post("/webhook", (req, res) => {
       payloadToSign,
     });
 
+    // Validates if the timestamp is within an acceptable timeframe
+    // For more information visit https://webhooks.fyi/security/replay-prevention
+    const isTimestampValid = validateTimestamp(revolutRequestTimestamp);
+
+    if (!isTimestampValid) {
+      // The request timestamp is outside of the tolerance zone.
+      return res.status(403).send("Timestamp outside the tolerance zone");
+    }
+
     if (isSignatureValid) {
+      const order = orders.getOrderByRevolutId(req.body["order_id"]);
```

Comment:
in real world it would be a DB call, should we do it here before sending 200?

Reply:
Makes sense, moved all order processing inside the worker